package.json

```diff
@@ -22,9 +22,11 @@
   "main": "./build/get-own-property-symbols.node.js",
   "scripts": {
     "build": "make",
-    "test": "node test/.test.js"
+    "test": "node test/.test.js",
+    "typecheck": "google-closure-compiler --js=src/*.js --checks_only --module_resolution=node --compilation_level=ADVANCED"
   },
   "devDependencies": {
+    "google-closure-compiler": "^20180910.0.0",
```

Comment:
I have no interest in depending on closure compiler.

Reply:
I added it to have a means of typechecking to prevent further issues. Will revert if you want to.

Reply:
Since it hasn’t actually found any real issues, i don’t think there’s any value in having it.